nucypher_ops/playbooks/include/update_existing_ursula.yml

```diff
@@ -22,4 +22,11 @@
         source: pull
         force_source: yes
 
+    - name: Run config migrations
+      become: yes
+      become_user: nucypher
+      command: "docker run -v /home/nucypher:/root/.local/share/ \
+        -it {{ docker_image | default('nucypher/nucypher:latest') }} \
+        nucypher ursula config migrate"
```

Comment:
After closer examination of the changeset in nucypher, this command requires the `--config-file <FILEPATH>` argument to be passed in order to certainly avoid an interactive prompt.